content/blog/2019/corda-simple-broadcast-transaction/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Broadcasting a transaction to external organisations
+date: "2019-05-29"
+published: true
+tags: [corda, kotlin, dlt, distributed ledger technology, blockchain]
+cover_image: ./corda-concert.png
+github_url: https://github.com/lankydan/corda-broadcast-transaction
+---
+
+Corda does not globally broadcast transactions between organisations/nodes by default. Privacy and not sharing unnecessary data is a central component of Corda's design. Compared to other DLT (Distributed Ledger Technology) platforms and blockchains, this is indeed a big difference. But, this statement comes with one fallacy. There is a belief that Corda cannot broadcast transactions to external nodes at all. This is simply wrong. It may not be the default behaviour but it is definitely within Corda's capabilities. In this post, I will demonstrate the small amount of code required to send a transaction to any desired node.
```

Comment:
Maybe you can start with.... there is a misconception that code cannot "broadcast" data... but this doesn't make sense ... because if you can send point to point messages ... then we can compose many point to point messages together to send the same data to multiple parties.

Perhaps also we need to define whaty broadcast actually means. No one actually "broadcasts" anything.. what other platforms really do is send data via point to point messages but orchestrate them in such a way to form a gossip protocol. Corda can do the same thing!

Reply:
Considering replacing the intro with

There is a misconception that Corda cannot _broadcast_ data across a network. This is simply wrong. In fact, Corda can send anything between nodes in a network. What Corda does not do, is share unnecessary data (transactions) with nodes that have nothing to do with an individual interaction. Privacy by default is a central component of Corda's design. Compared to other DLT (Distributed Ledger Technology) platforms and blockchains, this is indeed a big difference. Sharing data with non-transacting parties might not be part of Corda's default behaviour, but it is definitely within its capabilities. In this post, I will demonstrate the small amount of code required to send a transaction to any desired node.